docs/content/getting-started/_index.md

```diff
@@ -92,17 +125,155 @@ class App extends Component {
 
-```html
-  <pfe-card color="darkest">
-    <h2 slot="header">Dark card</h2>
-    <p>Lorem ipsum.</p>
-    <pfe-cta color="complement" priority="primary" on="dark" slot="footer">
-        <a href="#">Important call-to-action</a>
-    </pfe-cta>
-  </pfe-card>
+#### General
+
+  Priority & variant attributes change style styles globally:
+
+  ```
+  <pfe-cta pfe-priority="primary">...</pfe-cta>
+  <pfe-tabs pfe-variant="earth">...</pfe-tabs>
+  ```
+
+  **Full list:**
+
+  *   `pfe-priority`
+  *   `pfe-variant`
+
+
+#### Specific
+
+Color or align attributes change specific properties
+
+```
+<pfe-cta pfe-color="accent">...</pfe-cta>
 ```
 
 
+#### **Full list:**
+
+*   `pfe-align`
+*   `pfe-color`
+*   `vertical`
+*   `horizontal`
+
+
+## 5. Use CSS variables as needed
+
+CSS variables are subject to the normal cascade, so consider where you want these overrides to propogate.
+
+
+#### Page-level CSS, [theme](https://static.redhat.com/libs/redhat/redhat-theme/2/advanced-theme.css) variables
+
+Theme variables will impact all components on the page where this CSS is loaded.
+
+
+```
+// your-page.css
+:root {
+    --pfe-theme--color--ui-accent: green;
+    --pfe-theme--color--surface--darker: navy;
+}
+```
+
+
+_Note_: overriding local variables (i.e., --pfe-cta--foo) will not work at this level.  Those overrides can only be done on the component via tag name, class, or ID.
+
+
+#### Page-level CSS, component variables
+
+
+```
+/* your-page.css */
+pfe-cta {
+    --pfe-cta--BackgroundColor: green;
+}
+pfe-band {
+    --pfe-band--Padding--vertical: 34px;
+}
+```
+
+
+
+### Component-level inline CSS
+
+As a last resort, you may choose to override variables with inline styles. This could be desirable if you only need one component to deviate from the design system. Note that this incurs some level of risk, especially related to colors, as you are opting out of the color combinations in the system.
+
+
+```
+<pfe-cta style="--pfe-cta--BackgroundColor:pink">
+```
+
+
+## 6. Victory!
+
+Be sure to browser test within your own site or application. If you run into any trouble, please file an issue in the [issue queue](https://github.com/patternfly/patternfly-elements/issues).
+
+<hr/>
+
+
+## FAQ
+
+### Should I use on=dark or color=darkest on my container? What's the difference?
+
+*   `on=dark` is being deprecated. Instead, custom classes already on the page should set broadcast values.
+*   The original goal for `on=dark` was all about context. But instead of having to provide another attribute somewhere, the card or a band or another container will inform any nested components that the text color needs to change through the set of broadcast variables. Existing broadcast vars (defined in `$BROADCAST-VARS`)  include: `text`, `ui-link`, `ui-link--hover`, `ui-link--visited`, `ui-link--focus`.
```

Comment:
Think this should read:
The original goal for `on=dark` was all about context, but instead of having to provide another attribute somewhere, the card, a band, or another container will inform any nested components that the text color needs to change through the set of broadcast variables.

Reply:
updated, thx!